Index: src/main/java/ru/nurullin/spring/Person.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.nurullin.spring;\r\n\r\npublic class Person {\r\n    private String name;\r\n    private int age;\r\n\r\n    public Person(String name, int age) {\r\n        this.name = name;\r\n        this.age = age;\r\n    }\r\n\r\n    public String getName() {\r\n        return name;\r\n    }\r\n\r\n    public int getAge() {\r\n        return age;\r\n    }\r\n\r\n    public void setName(String name) {\r\n        this.name = name;\r\n    }\r\n\r\n    public void setAge(int age) {\r\n        this.age = age;\r\n    }\r\n\r\n    @Override\r\n    public String toString() {\r\n        return \"Person [name=\" + name + \", age=\" + age + \"]\";\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/ru/nurullin/spring/Person.java b/src/main/java/ru/nurullin/spring/Person.java
--- a/src/main/java/ru/nurullin/spring/Person.java	
+++ b/src/main/java/ru/nurullin/spring/Person.java	
@@ -1,5 +1,8 @@
 package ru.nurullin.spring;
 
+import org.springframework.stereotype.Component;
+
+@Component
 public class Person {
     private String name;
     private int age;
